src/main/java/com/salesforce/datacloud/jdbc/core/DataCloudConnection.java

```diff
@@ -173,6 +175,24 @@ private DataCloudPreparedStatement getQueryPreparedStatement(String sql) {
         return new DataCloudPreparedStatement(this, sql, new DefaultParameterManager());
     }
 
+    /**
+     * Use getQueryStatus to determine if your query is "ready" then use this to get a collection of rows.
+     * When using {@link RowBased.Mode#FULL_RANGE} this method is not responsible for calculating the offset near the end of available rows,
+     * you must calculate the correct "pages" of offset and limit.
```

Comment:
I don't understand what this part of the comment is supposed to tell me. And if I don't get it, I don't think our users will understand it, either

Reply:
I have changed this comment as well. Please take a look. @vogelsgesang